test/integration/retryable-writes/retryable_writes.spec.test.ts

```diff
@@ -40,6 +57,14 @@ describe('Legacy Retryable Writes Specs', function () {
 
         await utilClient.close();
 
+        if (
+          this.configuration.topologyType === 'Sharded' &&
+          lt(this.configuration.version, '4.4.0')
+        ) {
```

Comment:
We also need to check the description, right? we don't want to skip all tests on <4.4 mongoses.

Also - we usually attach a skip reason:

```typescript
this.currentTest.skipReason = ...
this.skip();
```